src/pages/careers/locations/_location-layout-new-offices.tsx

```diff
@@ -37,6 +37,8 @@ const SecondStyledHeader = styled(Header)`
     margin-bottom: 0 24px;
 `
 const StyledLinkButton = styled(LinkButton)`
+    padding: 15px 24px;
+    border-radius: 6px;
     font-size: 20px;
     line-height: 30px;
```

Comment:
We have `hero` props added to button. I believe it is similar styling if we use that prop? If so,  we can omit this extension of `LinkButton`.